pwa/src/templates/home/HomeTemplate.tsx

```diff
@@ -16,6 +16,20 @@ export const HomeTemplate: React.FC = () => {
   const queryClient = new QueryClient();
   const _useDashboardCards = useDashboardCards(queryClient);
   const getDashboardCards = _useDashboardCards.getAll();
+  const deleteDashboardCard = _useDashboardCards.createOrDelete();
+
+  const handleDeleteDashboardCard = (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent<HTMLButtonElement>,
+    dashboardCardId: string,
+  ) => {
+    e.stopPropagation();
+
+    const confirmDeletion = confirm("Are you sure you want to delete this object?");
```

Comment:
I think we can make this prompt more specific, e.g.: "Are you sure you want to delete this card?"